### DISCUSSION TITLE
Server Error after update

### DISCUSSION THREAD

**Discussion by arne1921KF** [Nachtzuster/BirdNET-Pi#46] [Q&A]
I updated Friday night, and can't reach my installation on the LAN since then. I get a HTML 500. 

I can login to the Raspi's shell, and BirdNet-Pi is running - I still get the detection notifications via apprise.

Any ideas what's up? I'm a bit at a loss what happened.

**Comment by Nachtzuster** [comment]
Moved to Issues

**Comment by arne1921KF** [comment]
Happy to report I found the culprit: hardware. 

I had to change the power supply last week, and apparently the power it supplies isn't enough for updating, or doing much else. When I found my adapter to micro-HDMI, I noticed the Pi rebooted every now and then when. htop didn't show high loads, but when it did, a reboot happened, repeatedly.

Anyway, going to the shell and calling update_birdnet.sh seems to have helped. The web server is again up and running, and detection seems to have been up and running most of the time.
